presto-hive-hadoop2/src/test/java/io/prestosql/plugin/hive/TestHiveHadoop2Plugin.java

```diff
@@ -65,6 +67,46 @@ public void deinitializeRubix()
         CachingFileSystem.deinitialize();
     }
 
+    @Test
+    public void testCreateConnector()
+    {
+        Plugin plugin = new HiveHadoop2Plugin();
+        ConnectorFactory factory = getOnlyElement(plugin.getConnectorFactories());
+        // simplest possible configuration
+        factory.create("test", ImmutableMap.of("hive.metastore.uri", "thrift://foo:1234"), new TestingConnectorContext()).shutdown();
+    }
+
+    @Test
+    public void testThriftMetastore()
+    {
+        Plugin plugin = new HiveHadoop2Plugin();
+        ConnectorFactory factory = getOnlyElement(plugin.getConnectorFactories());
+        factory.create(
+                "test",
+                ImmutableMap.of(
+                        "hive.metastore", "thrift",
+                        "hive.metastore.uri", "thrift://foo:1234"),
+                new TestingConnectorContext())
+                .shutdown();
+    }
+
+    @Test
+    public void testGlueMetastore()
```

Comment:
I am not convinced that it adds useful coverage, as we are not able to assert anything meaningful.

Reply:
this covers guice bindings. if not useful as a test itself, it is useful for development
(unless i forget to run the test...)

Reply:
in fact, there is currently no other test verifying GlueMetastoreModule works, as `TestHiveGlueMetastore` instantiates the metastore class directly

Reply:
I guess product tests which use Glue is more reasonable way of testing if GlueMetastoreModule works. Yet it is not something you want to run often during development. Feel free to leave it if you found it useful

Reply:
i agree that we should have them -- https://github.com/prestosql/presto/issues/5426